server/backend.go

```diff
@@ -1,9 +1,12 @@
 package server
 
-// Router paths
 var (
+	// Router paths
 	pathStatus            = "/eth/v1/builder/status"
 	pathRegisterValidator = "/eth/v1/builder/validators"
 	pathGetHeader         = "/eth/v1/builder/header/{slot:[0-9]+}/{parent_hash:0x[a-fA-F0-9]+}/{pubkey:0x[a-fA-F0-9]+}"
 	pathGetPayload        = "/eth/v1/builder/blinded_blocks"
+
+	// Relay Monitor paths
+	pathAuctionTranscript = "/eth/v1/relay-monitor/transcript"
```

Comment:
TBD whether the `/eth/` namespace should be reused (i think perhaps not). cc/ @ralexstokes